src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -342,7 +343,7 @@ private void testAddAction() throws Exception {
         newSession.setResultsVisibleFromTime(Const.TIME_REPRESENTS_NEVER);
         newSession.setGracePeriod(25);
         newSession.setInstructions(instructions);
-        newSession.setTimeZone(-2);
+        newSession.setTimeZone(TimeHelper.convertToZoneId(-2));
```

Comment:
Just use something like `ZoneId.of("-02:00")`? `convertToZoneId` should be removed eventually.